atomic

```diff
@@ -444,6 +444,7 @@ if __name__ == '__main__':
     scanp.add_argument("--scan_type", default=None, help=_("define the intended scanner"))
     scanp.add_argument("--list", action='store_true', default=False, help=_("List available scanners"))
     scanp.add_argument("--verbose", action='store_true', default=False, help=_("Show more output from scanning container"))
+    scan_group.add_argument("--rootfs", nargs='*', help=_("One or more paths to scan"))
     scan_group.add_argument("--all", default=False, action='store_true', help=_("scan all images (excluding intermediate layers) and containers"))
```

Comment:
I think it would be better to call --rootfs multiple times then attempt to deal with multiple paths on the same command.